nodecfg/node_config.go

```diff
@@ -418,6 +418,7 @@ func migrateNodeConfig(tx *sql.Tx) error {
 func loadNodeConfig(tx *sql.Tx) (*params.NodeConfig, error) {
 	nodecfg := &params.NodeConfig{}
 
+	keystoreDir := "" // TODO: remove this from db
```

Comment:
Can't we just remove it from the SQL select query?

Reply:
Ofc we can, but will still live in the table. This way reminds us to fully clean it, once.